Dockerfile

```diff
@@ -10,77 +9,32 @@
 # Description: Docker container image recipe
 
 FROM debian:stable
-MAINTAINER Fossology <fossology@fossology.org>
-WORKDIR /fossology
 
-ENV _update="apt-get update"
-ENV _install="apt-get install -y --no-install-recommends"
-ENV _cleanup="eval apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*"
-
-RUN set -x \
- && $_update && $_install \
-       lsb-release curl php5 libpq-dev libdbd-sqlite3-perl libspreadsheet-writeexcel-perl postgresql-client \
-       sudo \
-       # for standalone mode:
-       postgresql \
- && $_cleanup
+MAINTAINER Fossology <fossology@fossology.org>
 
-ADD utils/fo-installdeps utils/fo-installdeps
-ADD install/scripts/php-conf-fix.sh install/scripts/php-conf-fix.sh
-ADD utils/install_composer.sh utils/install_composer.sh
-RUN set -x \
- && $_update \
- && /fossology/utils/fo-installdeps -e -y \
- && $_cleanup \
- && /fossology/install/scripts/php-conf-fix.sh --overwrite \
- && /fossology/utils/install_composer.sh
+WORKDIR /fossology
+COPY . .
 
-ADD . .
-RUN chmod +x /fossology/docker-entrypoint.sh
-RUN set -x \
- && make install \
- && make clean
+RUN apt-get update && \
+    apt-get install -y lsb-release sudo postgresql php5-curl libpq-dev libdbd-sqlite3-perl libspreadsheet-writeexcel-perl && \
+    /fossology/utils/fo-installdeps -e -y && \
+    rm -rf /var/lib/apt/lists/*
 
-RUN set -x \
- && /usr/local/lib/fossology/fo-postinstall --common \
- && mkdir -p /srv/fossology/repository/
+RUN curl -sS https://getcomposer.org/installer | php && \
+    mv composer.phar /usr/local/bin/composer
 
-################################################################################
-# scheduler related
-RUN /usr/local/lib/fossology/fo-postinstall \
-        --agent \
-        --scheduler-only
+RUN /fossology/install/scripts/install-spdx-tools.sh
 
-RUN set -x \
- && mkdir -p /var/log/fossology \
- && chown -R fossy:fossy /var/log/fossology \
- && chgrp -R fossy /usr/local/etc/fossology/ \
- && chmod -R g+wr /usr/local/etc/fossology/ \
- && chown fossy:fossy /usr/local/etc/fossology/Db.conf
+RUN /fossology/install/scripts/install-ninka.sh
```

Comment:
It kinda is, because then docker is installing a broken dependency into FOSSology. If you then try to upload a file and check the ninka checkbox, it will fail 100%